### ISSUE TITLE
Graph - Possible to add iFrame authentication instead op popups?

### ISSUE THREAD

**Issue by pkmelee337** [pnp/pnpjs#336]
### Category
- [x ] Enhancement
- [ ] Bug
- [ ] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [ 1.2.3 ]

Please specify what version(s) of SharePoint you are targeting: [ SPO ]

### Expected / Desired Behavior / Question
Since a few weeks I have been busy implementing the pnpjs graph library in the webpart we have created so far. Apart from some issues, this is working fine so far and I really love the simple fluent api implementation you guys have created! Many thanks for that.

The thing people in our organization (and also our customers, but they do not have this version yet) really hate, is the login popup. In previous versions we catched this popup with a message on top of the page if the user would like to login.

In pnpjs Graph implementation the popup is coming up every time I am opening a new tab (even on the same session). This isn't necessary in my opinion, but I sadly did not have found the time yet to figure out what is causing this. What I would like to suggest is to implement some kind of Iframe authentication, and show the popup only if this iframe authentication failes. (or make this configurable) Most people do not have to login due to single signon, so the popup is only retrieving an access token to close the popup immediatly after this has succeeded. Within an Iframe I think this is also possible, and people won't have to see an strange popup open and close (which is also blocked on most browser by default)

**Comment by patrick-rodgers** [comment]
Closing this as duplicate of #335.
